eng/common/build.ps1

```diff
@@ -13,6 +13,7 @@ Param(
   [switch] $rebuild,
   [switch] $deploy,
   [switch][Alias('t')]$test,
+  [switch] $codeCoverage,
```

Comment:
I was planning for this to be a target rather than a property. I do have some changes for Arcade, I'll work with them to figure out the right pattern.